po/pt_BR.po

```diff
@@ -2,18 +2,24 @@
 # Copyright (C) 2005-2006 Giuseppe Torelli.
 # This file is distributed under the same license as the xarchiver package.
 # Automatically generated, 2006.
-# Adriano Winter Bess <adriano@xfce.org>, 2006.
-# Og Maciel <ogmaciel@gnome.org>, 2008, 2009.
+# Volunteer translators of the Brazilian Portuguese language (pt_BR):
+# Tradudores voluntários do idioma Português do Brasil (pt_BR):
```

Comment:
It is not your responsibility to make changes here, except to add or correct **your** name, email address, and year.